.tekton/cache-pull-request.yaml

```diff
@@ -20,8 +20,11 @@ spec:
       value: cache/src/main/docker/Dockerfile.all-in-one
   pipelineRef:
     name: docker-build
-    bundle: quay.io/redhat-appstudio/hacbs-core-service-templates-bundle:latest
+    bundle: quay.io/redhat-appstudio/build-templates-bundle:latest
```

Comment:
they got rid of `latest` tag @Michkov 

it is now `devel`

see https://github.com/redhat-appstudio/jvm-build-service/pull/302

Reply:
I've reopened 302 for comparison